examples/shouldi/shouldi/cargo_audit.py

```diff
@@ -0,0 +1,55 @@
+import json
+import asyncio
+from typing import Dict, Any
+
+from dffml.df.base import op
+from dffml.df.types import Definition
+
+package_src_dir = Definition(name="package_src_dir", primitive="str")
+cargo_audit_output = Definition(
+    name="golangci_lint_output", primitive="Dict[str, Any]"
+)
+
+
+class CargoAuditError(Exception):
+    """
+    Raised when cargo-audit fails
+    """
+
+
+async def run_cargo_build(pkg_input: str):
+
+    new_proc = await asyncio.create_subprocess_exec(
+        "cargo",
+        "build",
+        "--release",
+        cwd=pkg_input,
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.PIPE,
+    )
+    stdout, stderr = await new_proc.communicate()
+    if stderr == 0:
```

Comment:
```suggestion
    if stderr:
```

`stderr` is a [bytes](https://docs.python.org/3/library/stdtypes.html#bytes) object. It will never be equal to 0. However, if it is of length 0, it will evaluate to `False` in an if statement, therefore we can check with this ^ `if`.